codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -646,9 +647,10 @@ private LastHttpContent readTrailingHeaders(ByteBuf buffer) {
                         b.append(currentLastPos)
                          .append(lineTrimmed);
                         current.set(lastPos, b.toString());
-                    } else {
-                        // Content-Length, Transfer-Encoding, or Trailer
                     }
+                    //else {
+                    // Content-Length, Transfer-Encoding, or Trailer
```

Comment:
I realize you didn't introduce this comment but is this comment necessary? Can we make the comment more meaningful or just remove it?

@trustin - WDYT [1]

[1] https://github.com/doom369/netty/commit/4ce65f36212e7a97b902958ad6c56611091c83f5#diff-bba2e269877a9cc5f6b8ad204b1c89e5R451